apps/dashboard/config/environments/test.rb

```diff
@@ -55,6 +56,9 @@
   # Annotate rendered view with file names.
   # config.action_view.annotate_rendered_view_with_filenames = true
 
+  # Raise error when a before_action's only/except options reference missing actions
+  config.action_controller.raise_on_missing_callback_actions = true
```

Comment:
Tests are failing because of this right here. I think instead of setting this to `false` we should likely look into how we can refactor these callbacks to (a) be faster so we don't do everything everywhere and (b) to avoid this issue here.